### PR TITLE
fix: add spring-security extension metadata

### PR DESCRIPTION
**Pull request by aureamunoz**
I think I missed that when adding the spring-security extension.
WDYT @geoand @michalszynkiewicz @gsmet ?
Related to #5225

### CODE REVIEW

File: extensions/spring-security/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: "Quarkus Extension for Spring Security API"
+metadata:
+  keywords:
+  - "spring-security"
+  - "spring"
```
Comment (gsmet) [inline extensions/spring-security/runtime/src/main/resources/META-INF/quarkus-extension.yaml]: Maybe add `security` as a keyword too.

**Review by gsmet — CHANGES_REQUESTED** [review]
Looks good, thanks for taking care of that, I added a minor comment.

**Comment by gsmet** [comment]
@aloubyansky @maxandersen any chance we could fail if one extension is missing the metadata?

File: extensions/spring-security/runtime/src/main/resources/META-INF/quarkus-extension.yaml (hunk 1 above)
Comment (aureamunoz) [reply extensions/spring-security/runtime/src/main/resources/META-INF/quarkus-extension.yaml]: yep, good idea.

**Comment by aloubyansky** [comment]
We surely can fail if no descriptor is provided. OTOH, I like the idea of not requiring a metadata and generating the basic one as the default. It'd be one of the requirements however to have a detailed enough metadata to be accepted in the platform.

**Comment by gsmet** [comment]
@aloubyansky any chance it could be optional and we could enforce that in the Quarkus and Platform builds?

**Comment by aloubyansky** [comment]
I think so. However it actually has to be reviewed by somebody accepting
the extension, it's not something that can be relied upon blindly.

**Comment by gsmet** [comment]
@aloubyansky I think if the file is there, people will review it properly. But I'm not sure they will think of it if it's missing (we have a good example here).

File: extensions/spring-security/runtime/src/main/resources/META-INF/quarkus-extension.yaml (hunk 1 above)
Comment (gsmet) [reply extensions/spring-security/runtime/src/main/resources/META-INF/quarkus-extension.yaml]: I just force-pushed an amended commit with the `security` keyword added. Let's wait for CI.

**Review by gsmet — APPROVED** [review]

**Comment by maxandersen** [comment]
we definitely could do a "platform-lint" that could warn/errror on issues; especially something we could enforce in platform and at least in quarkus hosted extensions.

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**